elegant/gui/general_pose_annotator.py

```diff
@@ -0,0 +1,91 @@
+import freeimage
+import pathlib
+import pickle
+from PyQt5 import Qt
+import numpy
+from functools import partial
+
+from ris_widget import ris_widget
+from ris_widget import shared_resources
+
+from elegant.gui import pose_annotation
+from elegant import worm_widths
+from elegant import worm_spline
+
+from zplib.image import colorize
+
+class GeneralPoseAnnotator():
+    def __init__(self, rw):
+        self.ris_widget = rw
+        width_estimator = worm_widths.WidthEstimator.from_default_widths(pixels_per_micron=1/1.3)
```

Comment:
pixels per micron probably ought to be an option, as well as a command-line option.